usr/bin/revpi-btuart

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+# referenced pi-bluetooth, bluetooth attach for Revolution Pi BT
+. /usr/bin/revpi-common
+
+HCIATTACH=/usr/bin/hciattach
+TTYDEVNAME="$1"
+
+revpi_dev_type
+DTYPE=$?
+
+if [ "0" = $DTYPE ]; then
+  OPTS=""
+  TYPEID="any"
```

Comment:
This should be bcm43xx see here https://github.com/RPi-Distro/pi-bluetooth/blob/6f6386e91e33966d3c4a3cfee72d61e14c6a11ef/usr/bin/btuart#L25